src/main/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/UpdateManyCommandResolver.java

```diff
@@ -77,6 +77,7 @@ private FindOperation getFindOperation(CommandContext commandContext, UpdateMany
         objectMapper,
         null,
         0,
-        0);
+        0,
```

Comment:
Same as DeleteManyCommandResolver, can we use FindOperation.unsorted() instead of direct constructor.

Reply:
Yes of course..

Reply:
Done.